### ISSUE TITLE
Viewport setting - cull backfaces

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/editor#1154]
This is a feature request to add an option to cull backfaces in the Editor viewport, similar to the wireframe mode. I would like not to change the actual material properties. This will make it a bit easier when working in close environments.

Would be nice to be able to adjust near/far clip of the camera as well, but maybe not easy from UX side.

https://github.com/playcanvas/editor/assets/5677782/a3f1b976-2478-4313-bff0-723f6cf412cc

**Comment by mvaligursky** [comment]
Hi .. I'm not entirely sure what you mean here. By default, the backfaces are culled. Are you hoping to not cull them? That could be useful with wireframe to see them, but with a solid rendering, you would not see them as they face away?

**Comment by LeXXik** [comment]
Yeah, by default it is. But if the materials don't cull it, then the viewport will render back faces when the camera goes inside a mesh. So, the only way is to change the actual material properties. I would like to avoid it touching stuff the artists set, as I will definitely forget to revert it back later.

I tried the wireframe, and though it solves the original issue of losing object focus, the viewport becomes a bit too busy and it becomes hard to do a precise position, as you can't see if you are clipping a mesh:

![image](https://github.com/playcanvas/editor/assets/5677782/dd2d11b8-072b-4eda-ae1e-4b28884a7548)

**Comment by mvaligursky** [comment]
Yep that makes sense now, so your materials are set up as double sided.
